cli/release.go

```diff
@@ -149,3 +156,31 @@ func runReleaseAddDocker(args *docopt.Args, client *controller.Client) error {
 
 	return nil
 }
+
+func canonicalizeDockerURI(rawurl string) (string, error) {
+	log.Println("Validating Docker image URI.")
```

Comment:
Sorry, I meant the help text (i.e. [this](https://github.com/flynn/flynn/blob/master/cli/release.go#L63-L64)) needs updating to include this log message.